vrk-kpa/suomidigi
# SUOPA-894 Blog post

To test: 
- `make fresh`
- Log in as an admin and clear the caches.
- Go to https://suomidigi.docker.sh/user/register with an **incognito browser**
   - Create a user and log in with the user with `drush uli --mail=` and the email you used
- Go to https://suomidigi.docker.sh/user and find the link what says "Luo blogikirjoitus" and click on it
   - Try to create a blog post and see that the node form resembles the DoD
   - Save the node with a state: "Submit for publication"
- Open the browser with admin logged in and go to https://suomidigi.docker.sh/admin/content/moderated
   - Find the freshly created blog post and edit it. 
   - Scroll down to the bottom of the node and switch the state to Published and save the node.
   - Check that the node gets published.
- Go back to the incognito browser and try to edit the node. 
   - Check that you cannot save the node as published state.
   - You should be able to save the node with changed data, but it'll be a new revision and won't override the actual node until the admin or editor in chief has published it.
